assets/src/components/mapPage/vehiclePropertiesCard.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { ReactNode, useId } from "react"
 import { joinTruthy, joinClasses } from "../../helpers/dom"
 import { useCurrentTimeSeconds } from "../../hooks/useCurrentTime"
-import { useNearestIntersection } from "../../hooks/useNearestIntersection"
+import { useNearestIntersectionFetchResult } from "../../hooks/useNearestIntersection"
```

Comment:
Did this PR pull in some unrelated changes? Would a rebase fix this?

Reply:
Therrrrrrre we go. Not sure how I got things entangled

Reply:
Git'll do that to ya!